### PR TITLE
chore: Fix formatting of SSH docs markdown table

### PR DESCRIPTION
**Pull request by Gabrielopesantos**
Small PR to fix the formatting of a markdown table in the SSH docs.

## Target Release

1.14.7

### CODE REVIEW

File: website/content/api-docs/secret/ssh.mdx [hunk 1]
```diff
@@ -590,7 +590,7 @@ This endpoint allows submitting the CA information for the secrets engine via an
 key pair. _If you have already set a certificate and key, they will be
 overridden._
 
-| Method | Path             |
+| Method | Path             |                            |
```
Comment (cipherboy) [inline website/content/api-docs/secret/ssh.mdx:593]:
```suggestion
| Method | Path             | Content-Type               |
```

perhaps for consistency with the `GET /ssh/public_key` endpoint.

**Review by cipherboy — APPROVED** [review]
I think this is the only update that needs to be done to this document. Thanks!

**Comment by Gabrielopesantos** [comment]
I don't think this change requires a changelog but let me know, I can add one.

**Comment by cipherboy** [comment]
@Gabrielopesantos Looks like the [DCO is missing](https://github.com/openbao/openbao/blob/main/CONTRIBUTING.md#developer-certificate-of-origin-dco-sign-off-policy) from the first commit, mind rebasing to fix that?

**Review by JanMa — APPROVED** [review]

**Comment by JanMa** [comment]
Thanks @Gabrielopesantos

**Pull request merged**